### DISCUSSION TITLE
What if there were more options for splitting long words (syllabic and affix)?

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#30] [Q&A]
Would it introduce fatal flaws into the system, such as lead to more conflicts or being impossible to enter certain phrases? 

(I know that adding what I suggest below might be out of the scope of the lapwing approach, just would like to hear an opinion).

* if the the affixes were applicable to grammatical root morphemes - not only to the semantically related standalone English words. 
* if in addition - pure syllabic splitting were allowed
* where appropriate - for the longest words allow more outlines with "hanging" vowels - to elaborate on the syllabic splitting
* have exhaustive variants of matching vowel outlines - where appropriate

The above I think would make it easier to enter longer words and it would make the use of etymology optional, but whoever likes it would be able to take advantage of it.

This would also minimize potentialy subjective interpretation of the relatedness of words, which is used in Lapwing to decide whether to use suffixes or not - this by the way introduces an implicit requirement for some knowledge of etymology.

(Chapter 13 argues that words "dent" and "trident" are not related - but they might be related via the common ancestor of English and Latin - where DENT means TOOTH and word and older English dynt would have a figurative meaning of "tooth mark"; there is another set of words that are definitely related to dent - "indent" and its relatives. There is a similar argument about fuse and confuse - to me these words feel related. Fuse means to melt and connect parts, confuse - mix up "melt" the meanings. So relatedness of words may be a subjective notion.)

Thanks! Btw after 3 weeks on syllabic splitting, I got past 90% accuracy on the chapter 15 test; my first attempt was like 30%.

**Comment by aerickt** [comment]
I want to start off by saying that your points are completely valid and I'm not criticizing the ideas themselves. Below is just my two cents on why Lapwing is kinda the way that it is.

One of my motivations behind making Lapwing was that it should provide the user a set of rules that allow consistently writing any word in a way that's intuitive. However, as I based Lapwing off of Plover, I wanted to keep affix strokes. One of my main gripes with Plover theory's default dictionary is that it used affixes quite haphazardly. Sometimes it seemed like they made sense semantically (Lapwing-esque) and other times they seemed to be used completely orthographically. I decided that most people would about "standalone words", so I decided to go with that. Of course, as you've noted, it can still be pretty ambiguous at times. For some ambiguous words where it may not be immediately obvious that an affix can be used, there is also a syllabic split present.

I think for most people that has worked pretty well, favouring syllabic splits and using affixes only when it is very obvious. For Lapwing, I don't think it would be a good idea to add extra splits for a few reasons. First of all, the lookup tool is already quite overloaded for a lot of outlines, and adding more would make the problem worse. Furthermore, the tool is limited to showing at most 10 outlines. A lot of learners use the lookup tool for learning new words and steno in general. Second of all, having so many different ways of splitting a word may hinder the speedbuilding process. I believe it's better to learn one method and stick to it for the majority of the time. Finally, adding all of these variants and researching the various etymologies is just a monumental task for an individual like myself.

In general, steno is a very personal skill. Everyone has a differing opinion on what makes sense to them. Instead of tailoring a theory to fit everyone's ideas, I think it's better to tailor your own theory/dictionary to fit your ideas. No one should learn "stock Lapwing" without making any changes to the dictionary/theory. A lot of the speed you gain is from making exceptions and adding rules that make sense to you. That's how steno works traditionally; steno students will often be given a small dictionary of a few thousand entries, and will have to expand it in size to fifty or a hundred times larger.

However, if one disagrees with the underlying theory's core principles, then they may want to consider modifying it, or switching to a theory that aligns better with their intuition. I personally think in your case this might be the best course of action. Unfortunately, I am not aware of a theory/dictionary that uses affixes consistently as you've described. In fact, I'm not aware of another alternative to Lapwing/Plover/Platinum theory, which is quite problematic too. Ideally, there would be an abundance of steno theories and ideas for learners to choose from (maybe even a tool that can autogenerate dictionaries based on tweakable theory parameters).

While there isn't a steno theory that will align with everyone's intuition and accents, I do think some of Cocoa theory's ideas comes a lot closer (certainly when compared to Lapwing). There's no dictionary available for this theory yet, but the theory specifications page has a lot of cool concepts that may interest you: https://github.com/Kaoffie/cocoa-specs. In particular, Cocoa uses affixes much less, and its version of "syllabic splitting" can be used almost exclusively. If I created the Lapwing dictionary again, I would definitely consider this path. It's much easier from a dictionary building standpoint, and it removes a lot of the etymological subjectiveness that you've encountered.
